tests/validator/output/api/error_definitionuri_jsonschema3.json

```
@@ -0,0 +1,9 @@
+[
+  "[Resources.ApiDefinitionUriBucketEmpty.Properties.DefinitionUri.Bucket] Must not be empty",
+  "[Resources.ApiDefinitionUriBucketMissing.Properties.DefinitionUri] 'Bucket' is a required property",
+  "[Resources.ApiDefinitionUriBucketNotIntrinsic.Properties.DefinitionUri.Bucket] {'Not': 'Intrinsic'} is not of type 'string', 'intrinsic'",
+  "[Resources.ApiDefinitionUriBucketNotString.Properties.DefinitionUri.Bucket] 3 is not of type 'string', 'intrinsic'",
+  "[Resources.ApiDefinitionUriKeyEmpty.Properties.DefinitionUri.Key] Must not be empty",
+  "[Resources.ApiDefinitionUriKeyMissing.Properties.DefinitionUri] 'Key' is a required property",
+  "[Resources.ApiDefinitionUriKeyNotString.Properties.DefinitionUri.Key] {'Not': 'Intrinsic'} is not of type 'string', 'intrinsic'",
+]
```

I am surprise our new formatter didn't catch this!
```suggestion
]

```

---

https://github.com/aws/serverless-application-model/actions/runs/3491603658/jobs/5844448795

---

Need to run `make black` once